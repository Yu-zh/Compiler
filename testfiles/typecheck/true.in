int main(){
    int p = true;
    bool q = false;
    while(true){
        int a = 1;
    }
    return false;
    int r = f();
    int s = five();
    int a = 5;
    a = a + 1;
}
void f(){
    return;
}
bool five(){
    return 5;
}