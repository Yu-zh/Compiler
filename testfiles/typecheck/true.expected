Error at line 2: type does not check!
Error at line 8: type does not check!
Error at line 9: type does not check!